examples/tagging/private_tagging.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    mistral_output = mistral_program(user_comment="I hate this people they're lazy as fuck!")
</pre><p>emm could we use a better wording lol</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/9602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Okay, I'll change the phrasing to more appropriate one 😅